.gitignore

```diff
@@ -1,4 +1,5 @@
 !.gitignore
 .idea/
 db.sqlite3
-**/__pycache__/
\ No newline at end of file
+**/__pycache__/
```

Comment:
You can use default gitignore for python development like this one: https://github.com/github/gitignore/blob/master/Python.gitignore
And when you need to add smth, just add it to the bottom.